api/v1alpha1/argocd_types.go

```diff
@@ -702,6 +702,9 @@ type ArgoCDStatus struct {
 
 	// RepoTLSChecksum contains the SHA256 checksum of the latest known state of tls.crt and tls.key in the argocd-repo-server-tls secret.
 	RepoTLSChecksum string `json:"repoTLSChecksum,omitempty"`
+
+	// URL is the url for the hostname of the route/ingress
+	URL string `json:"url,omitempty"`
```

Comment:
Please rename this variable to `Host` and update the comment.  It is not really a URL. 
```
Host is the hostname of the Route or Ingress.
```